samples/azure-quantum/grover/README.md

```diff
@@ -1,14 +1,15 @@
 ---
 page_type: sample
-author: cgranade
```

Comment:
Elsewhere, we've been told that these keys are required; will they still work if they're scoped to `extensions`?

Reply:
They're required for topics that go to our usual OPS publishing, but not for the Samples Browser.  (Per the [sample metadata guidance](https://review.docs.microsoft.com/en-us/help/contribute/samples/process/onboarding?branch=master#supported-metadata-fields-for-readmemd) and poking around in many other Azure service samples).

My suspicion is that the Samples indexer doesn't like the metadata layout and isn't publishing the sample or indexing the azure-quantum service in the filter list. But I could be wrong.

Reply:
@cgranade - can we push this through and see if it fixes the publishing for these samples?